mlops/parallelm/mlops/mlops.py

```diff
@@ -254,6 +257,13 @@ def suppress_connection_errors(self, suppress):
         """
         SuppressException.set_suppress(suppress)
 
+    @property
```

Comment:
Move out test_mode from mlops it should be in mlcomp

Reply:
I don't see the way how it can be just moved to mlcomp. We don't have any mlcomp module.
Mlops is used everywhere and plumbed into Java.
Do you want to create a new module?